src/state/middleware/scigateway.middleware.test.tsx

```diff
@@ -54,6 +66,13 @@ describe('scigateway middleware', () => {
     },
   };
 
+  const loadDarkModePreferenceAction = {
+    type: LoadDarkModePreferenceType,
+    payload: {
+      preference: false,
```

Comment:
I thought the type of `LoadDarkModePreferencePayload` means this should be `darkMode: false`? Or am I misunderstanding